### DISCUSSION TITLE
[Resolved] `delay-{n}` does not appear to be working

### DISCUSSION THREAD

**Discussion by chrissm79** [nativewind/nativewind#1341] [General]
I have my animations with keyframes set at 0% and 100% with opacity from 0 to 1. Along with the fade animation class I'm adding a `delay-1000` with the intention of delaying the start of the fade in for 1 second, but the delay is not working. Am I using this correctly, that is that the intention of the `delay-{n}`? I also tried to add `delay-[1s]` and `delay-[1000]` but nothing seems to delay the animation. I'm using Expo v52 w/ react-native-reanimated 3.16.1.

**Comment by marklawlor** [comment]
`delay-{n}` sets a transition delay, not an animation delay

https://tailwindcss.com/docs/transition-delay
